haar_cascade/ImagePoint center
uint16 width
uint16 height
haar_cascade/ImagePoint[4] cornerPoints
